managed/services/agents/versioner_test.go

```diff
@@ -41,6 +41,8 @@ func TestSoftwareName(t *testing.T) {
 
 	for _, tc := range tests {
 		t.Run(string(tc.name), func(t *testing.T) {
+			tc := tc
```

Comment:
This should be done outside `t.Run`

Reply:
Great catch! I wonder why the linter doesn't capture such misses?